Firmware/menu.cpp

```diff
@@ -347,6 +349,11 @@ bool __attribute__((noinline)) menu_item_leave(){
     return ((menu_item == menu_line) && menu_clicked && (lcd_encoder == menu_item)) || menu_leaving;
 }
 
+bool menu_item_enter() {
+    menu_changed = false;
+    return !menu_changed;
```

Comment:
I suspect this is not correct. This always returns true. Will look at this when I have time.